neurodamus/node.py

```diff
@@ -1598,7 +1603,9 @@ def clear_model(self, avoid_creating_objs=False, avoid_clearing_queues=True):
         if not avoid_creating_objs and SimConfig.use_neuron and self._sonatareport_helper:
             self._sonatareport_helper.clear()
 
-        Node.__init__(self, None, None)  # Reset vars  # noqa: PLC2801
+        # Reset vars
+        Nd.init()
```

Comment:
I feel that we don't need `Nd.init()` here. It should be already set in `Node.__init__`. Can you try ?

Reply:
this moves away from the previous implementation. yes, I can try

Reply:
it looks like it works. I will merge if CI passes